src/core/components/parameter-row.jsx

```diff
@@ -212,7 +212,7 @@ export default class ParameterRow extends Component {
     let { schema } = getParameterSchema(param, { isOAS3 })
     let paramWithMeta = specSelectors.parameterWithMetaByIdentity(pathMethod, rawParam) || Map()
 
-    let format = param.get("format")
+    let format = param.get("format") || schema.get("format")
```

Comment:
```suggestion
    let format = schema.get("format")
```

Since `schema` holds the Schema Object for both 2.0 and 3.0, we can reuse it to keep the logic simpler!